standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Buffers.xml

```diff
@@ -24,7 +24,7 @@
     <member name="M:System.Buffers.ArrayPool`1.Rent(System.Int32)">
       <summary>Retrieves a buffer that is at least the requested length.</summary>
       <param name="minimumLength">The minimum length of the array.</param>
-      <returns>An array of type <see cref="T[]" /> that is at least <paramref name="minimumLength" /> in length.</returns>
+      <returns>An array of type T that is at least <paramref name="minimumLength" /> in length.</returns>
```

Comment:
Ditto.

Reply:
See answer. That's how we have it.
https://github.com/dotnet/dotnet-api-docs/blob/6f84948d2563a9b53a9cfb292d78a2caecacc2dd/xml/System.Buffers/ArrayPool%601.xml#L1